apps/wpcom-block-editor/src/default/features/rich-text.js

```diff
@@ -28,7 +41,7 @@ const ConnectedRichTextJustifyButton = compose(
 		return {
 			blockId: selectedBlock.clientId,
 			blockName: selectedBlock.name,
-			isBlockJustified: 'justify' === get( selectedBlock, 'attributes.align' ),
+			attributes: selectedBlock.attributes,
```

Comment:
Probably better to just pass "style" to avoid re-rendering the component if other attributes change.